demo/FontEnumerator.py

```diff
@@ -9,6 +17,11 @@ class TestPanel(wx.Panel):
     def __init__(self, parent, log):
         wx.Panel.__init__(self, parent, -1)
 
+        # Show how to add a private font to the application at runtime that
```

Comment:
Instead of updating an existing sample please add a new one. This is a new feature so it should be highlighted with a new sample.

The new sample could be just a simple panel with a static text stating something like, "The font used in the text below was dynamically loaded from {filename}." followed by another static text using the font.

Reply:
Ok. will add another demo for this.